homeassistant/components/yeelight/light.py

```diff
@@ -789,23 +803,30 @@ def set_scene(self, scene_class, *args):
 class YeelightColorLightSupport:
     """Representation of a Color Yeelight light support."""
 
+    _attr_supported_color_modes = {COLOR_MODE_COLOR_TEMP, COLOR_MODE_HS, COLOR_MODE_RGB}
+
     @property
-    def supported_features(self) -> int:
-        """Flag supported features."""
-        return SUPPORT_YEELIGHT_RGB
+    def color_mode(self):
+        """Return the color mode."""
+        color_mode = int(self._get_property("color_mode"))
+        if color_mode == 1:  # RGB
+            return COLOR_MODE_RGB
+        if color_mode == 2:  # color temperature
+            return COLOR_MODE_COLOR_TEMP
+        if color_mode == 3:  # hsv
+            return COLOR_MODE_HS
+        return COLOR_MODE_UNKNOWN
```

Comment:
Maybe add a warning logging here to inform the user in case an unexpected color mode is returned by the device?

Reply:
Makes sense, fixed in ff18010 + checked in tests